bee-node/bee-node/config.chrysalis-mainnet.toml

```diff
@@ -106,22 +106,31 @@ feature_proof_of_work = true
 white_flag_solidification_timeout = 2
 
 [snapshot]
+full_path         = "./snapshots/mainnet/full_snapshot.bin"
+delta_path        = "./snapshots/mainnet/delta_snapshot.bin"
+[[snapshot.download_urls]]
+full              = "https://chrysalis-dbfiles.iota.org/snapshots/hornet/latest-full_snapshot.bin"
+delta             = "https://chrysalis-dbfiles.iota.org/snapshots/hornet/latest-delta_snapshot.bin"
+[[snapshot.download_urls]]
+full              = "https://cdn.tanglebay.com/snapshots/mainnet/full_snapshot.bin"
+delta             = "https://cdn.tanglebay.com/snapshots/mainnet/delta_snapshot.bin"
+[snapshot.create]
+enabled           = false
 depth             = 50
 interval_synced   = 50
 interval_unsynced = 1000
-full_path   = "./snapshots/mainnet/full_snapshot.bin"
-delta_path  = "./snapshots/mainnet/delta_snapshot.bin"
-[[snapshot.download_urls]]
-full        = "https://chrysalis-dbfiles.iota.org/snapshots/hornet/latest-full_snapshot.bin"
-delta       = "https://chrysalis-dbfiles.iota.org/snapshots/hornet/latest-delta_snapshot.bin"
-[[snapshot.download_urls]]
-full        = "https://cdn.tanglebay.com/snapshots/mainnet/full_snapshot.bin"
-delta       = "https://cdn.tanglebay.com/snapshots/mainnet/delta_snapshot.bin"
 
 [pruning]
-enabled         = true
-delay           = 60480
-prune_receipts  = false
+[pruning.milestones]
+enabled                = true
```

Comment:
maybe "enable" instead `enabled` ?

Reply:
The Bee config should be consistent with the Hornet config as much as possible, so it doesn't cause confusion for people working with both nodes, hence, I just followed their naming conventions.